flink-streaming-java/src/test/java/org/apache/flink/streaming/api/functions/sink/filesystem/LocalStreamingFileSinkTest.java

```diff
@@ -484,131 +479,4 @@ public void testScalingDownAndMergingOfStates() throws Exception {
 			Assert.assertEquals(3L, counter);
 		}
 	}
-
-	@Test
-	public void testMaxCounterUponRecovery() throws Exception {
```

Comment:
Is this test not relevant anymore?

Reply:
It is relevant but it is subsumed by the `BucketsTest.testMergeAtScaleInAndMaxCounterAtRecovery()`.